tests/test_tools.py

```diff
@@ -16,109 +16,201 @@ def _setup_dirs(monkeypatch, user_filesystem):
     return home_dir
 
 
-def _run_tests(inputs, expected):
-    args = {"username": inputs[0], "email": inputs[1]}
-    expected_username, expected_email = expected
-    config = get_user_info(args)
+def _run_tests(cli_inputs, expected):
+    user_info = {"username": cli_inputs["cli_username"], "email": cli_inputs["cli_email"]}
```

Comment:
"cli_inputs" refer to function arguments passed to `get_user_info`. Not sure if this is the best name here but I want to distinguish them from user inputs via prompts ("inputs").

Reply:
Do we want to still have a helper function called `_run_tests` within each test func? 

The name `run_tests` to me is quite ambiguous - what test is it running? comparing username and email?

Reply:
Whoops missed this comment. Yeah I think we can keep this test  but maybe have a better name